pkgs/development/python-modules/quamash/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pytest, isPy3k, pyqt5, pyqt ? pyqt5 }:
+
+buildPythonPackage rec {
+  pname = "quamash";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "harvimt";
+    repo = "quamash";
+    rev = "version-${version}";
+    sha256 = "117rp9r4lz0kfz4dmmpa35hp6nhbh6b4xq0jmgvqm68g9hwdxmqa";
+  };
+
+  propagatedBuildInputs = [ pyqt ];
+
+  disabled = !isPy3k;
+
+  checkInputs = [ pytest ];
+  checkPhase = ''
+     pytest -k 'test_qthreadexec.py' # the others cause the test execution to be aborted, I think because of asyncio
```

Comment:
This is the best I could get. Just pytest results in something like
```
running install tests
============================= test session starts ==============================
platform linux -- Python 3.7.2, pytest-3.9.3, py-1.7.0, pluggy-0.8.0
rootdir: /build/source, inifile: tox.ini
collected 45 items

quamash/__init__.py ./nix/store/phl3z5qk8s4xnf74824navw5d0lsfka5-stdenv-linux/setup: line 1306:  8584 Aborted                 pytest
builder for '/nix/store/sbp7l7sm901hc7i26hxg6fcxj63x96hh-python3.7-quamash-0.6.1.drv' failed with exit code 134
error: build of '/nix/store/sbp7l7sm901hc7i26hxg6fcxj63x96hh-python3.7-quamash-0.6.1.drv' failed
```

I think it's because of `asyncio`. I'll leave the few tests that are left, it's better than nothing ¯\_(ツ)_/¯